src/app/modules/customer-management/components/projects-type/components/create-project-type/create-project-type.component.ts

```diff
@@ -63,9 +65,11 @@ export class CreateProjectTypeComponent implements OnInit, OnDestroy {
         id: this.projectTypeToEdit.id,
       };
       this.store.dispatch(new UpdateProjectType(projectType));
+      this.isHaveChanges.emit(this.haveChanges = false);
     } else {
       this.store.dispatch(new CreateProjectType({ ...projectTypeData, customer_id: this.customerId }));
       this.projectTypeForm.get('description').setValue('');
+      this.isHaveChanges.emit(this.haveChanges = false);
```

Comment:
The `obSubmit` function always output a `this.haveChanges = false`?

Reply:
I checked this line, is no necessary.